src/NUnitFramework/nunitlite/Options.cs

```diff
@@ -569,14 +573,17 @@ public OptionException(string message, string optionName, Exception innerExcepti
             _option = optionName;
         }
 
+#if !NET8_0_OR_GREATER
         protected OptionException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
             _option = info.GetString("OptionName");
         }
+#endif
 
         public string OptionName => _option;
 
+#if !NET8_0_OR_GREATER
 #if !NET6_0_OR_GREATER
```

Comment:
Doesn't `#if !NET6_0_OR_GREATER` imply `#if !NET8_0_OR_GREATER` ? 
Or is this to to anticipate when we remove the .NET6 target ?